Makefile

```diff
@@ -5,7 +5,7 @@ endif
 PKGS=$(shell go list ./... | grep -v /vendor)
 CI_PKGS=$(shell go list ./... | grep -v /vendor | grep -v test)
 FMT_PKGS=$(shell go list -f {{.Dir}} ./... | grep -v vendor | grep -v test | tail -n +2)
-SHELL_IMAGE=golang:1.8.3
+SHELL_IMAGE=golang:1.9.3
```

Comment:
Would this build with a newer version of golang or is`1.9.3` the latest that will work without changes?

Reply:
i did not test further, 1.9.* is probably the minimum upgrade to pass compiling, the e2e test based on kubicorn binary (not in this repo) passed after this change.

also, there is a discussion around this seems to confirm the theory:
https://github.com/kubernetes/test-infra/issues/7183